docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -0,0 +1,188 @@
+# Deploying OpenShift bare-metal workers on OpenStack cloud provider
+TBD introduction
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Conciderations whe deploying bare-metal workers](#conciderations-whe-deploying-bare-metal-workers)
+- [Deploying cluster with BM workers on tenant network deployed by the installer](#deploying-cluster-with-bm-workers-on-tenant-network-deployed-by-the-installer)
+- [Deploying cluster with BM workers on preexisting network](#deploying-cluster-with-bm-workers-on-preexisting-network)
+- [Deploy cluster with vm workers only and add BM workers after](#deploy-cluster-with-vm-workers-only-and-add-bm-workers-after)
+
+
+## Prerequisites
+* [Ironic bare metal service][2] is enabled an accessible through the [OpenStack Compute API][3].
```

Comment:
s/an/and/

Reply:
hmm. I thought that scenario is with vms and BMs on the same subnet.  is it meant to be with two different subnets? I guess regardless is up to the user to make sure BMs and vms can connect via network, but if they put workers on different subnets,  the security rules need to be modified. would the installer handle that correctly?  looks like perhaps a littel bit of testing there is required.